### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by oh6hay**
Content is as in other key repositories, eg. mattermost-mobile.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+----------------
+
+Mattermost has a mandatory upgrade policy, and updates are only provided for the latest release. Critical updates are delivered as dot releases. Details on security updates are announced 30 days after the availability of the update.
+
+For more details about the security content of past releases, see the [Security Updates](https://mattermost.com/security-updates/) page on the Mattermost website. For timely notifications about new security updates, subscribe to the [Security Bulletins Mailing List](https://about.mattermost.com/security-bulletin).
+
+Contributing to this policy
+---------------------------
+
+If you have feedback or suggestions on improving this policy document, please [create an issue](https://github.com/mattermost/mattermost-mobile/issues/new).
```
Comment (wiggin77) [inline SECURITY.md:25]:
```suggestion
If you have feedback or suggestions on improving this policy document, please [create an issue](https://github.com/mattermost/focalboard/issues/new/choose).
```

**Review by wiggin77 — CHANGES_REQUESTED** [review]

**Comment by chenilim** [comment]
@oh6hay, just following up on this. Sorry, I wan't sure if it needed my review - looks like Doug requested a change. Was there anything else? Should we link to this file from anywhere?

**Comment by oh6hay** [comment]
> @oh6hay, just following up on this. Sorry, I wan't sure if it needed my review - looks like Doug requested a change. Was there anything else? Should we link to this file from anywhere?

hi! thanks @wiggin77 for spotting the copypaste error in the SECURITY.md. @chenilim No it doesn't need to be linked from anywhere, SECURITY.md is a standard location for a security policy file. Thanks guys!

**Review by wiggin77 — APPROVED** [review]

**Review by chenilim — APPROVED** [review]
Thanks all!

**Pull request merged**